### DISCUSSION TITLE
Stopped working, no output?

### DISCUSSION THREAD

**Discussion by InconsolableCellist** [jackMort/ChatGPT.nvim#298] [Q&A]
I don't have much debugging information to go off of, but I had this plugin pointed to my own koboldcpp API, and it was working, then when starting it recently it simply sits at "loading" whenever I submit a prompt. How can I find more information about what's failing?

The "Issues" tab for this repo is also disabled and should be enabled for better issue tracking/search.

**Comment by jackMort** [comment]
issues are enabled, always ware, you can check the `:messages` for some errors
